### PR TITLE
(RE-8191) Remove java7 references from package builds

### PR DESCRIPTION
**Pull request by shrug**

### CODE REVIEW

**Comment by haus** [comment]
@shrug probably worth tweaking https://github.com/puppetlabs/ezbake/blob/master/resources/puppetlabs/lein-ezbake/template/foss/ext/build_defaults.yaml#L4-L5 as well?

File: resources/puppetlabs/lein-ezbake/template/foss/ext/build_defaults.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 packaging_url: 'git://github.com/puppetlabs/packaging.git --branch=master'
 packaging_repo: 'packaging'
-default_cow: 'base-trusty-i386.cow'
-cows: 'base-jessie-i386.cow base-precise-i386.cow base-trusty-i386.cow base-wheezy-i386.cow base-wily-i386.cow base-xenial-i386.cow'
```
Comment (mwaggett) [inline resources/puppetlabs/lein-ezbake/template/foss/ext/build_defaults.yaml]: don't we still build for wheezy?

Comment (shrug) [inline resources/puppetlabs/lein-ezbake/template/foss/ext/build_defaults.yaml]: For foss we'll have to stop as we're dropping java7 support

Comment (mwaggett) [inline resources/puppetlabs/lein-ezbake/template/foss/ext/build_defaults.yaml]: 😮

**Comment by shrug** [comment]
Needs https://github.com/puppetlabs/puppetlabs-debbuilder/pull/62 and https://github.com/puppetlabs/puppetlabs-rpmbuilder/pull/118

**Comment by underscorgan** [comment]
ping @shrug what's the status here?

**Comment by shrug** [comment]
This won't work with SLES-12 yet, blocked on updating mock on the builders and reverting https://github.com/puppetlabs/puppetlabs-modules/pull/6900

**Comment by shrug** [comment]
oops, now waiting on https://github.com/puppetlabs/puppetlabs-modules/pull/6971

**Comment by camlow325** [comment]
@shrug Since the related puppetlabs-modules PRs have been merged and we have two approvals on this PR are we good to go for merging this - and cutting a new ezbake release with it?

**Comment by shrug** [comment]
I believe we are now, yes

**Pull request merged**